tests/NHSD.BuyingCatalogue.Ordering.Services.UnitTests/OrderServiceTests.cs

```diff
@@ -0,0 +1,232 @@
+        }
+
+        [Test]
+        [InMemoryDbAutoData]
+        public static async Task GetOrder_ReturnsOrder(
+            [Frozen] ApplicationDbContext context,
+            Order order,
+            OrderService service)
+        {
+            context.Order.Add(order);
+            await context.SaveChangesAsync();
+
+            var expectedResult = order;
+
+            var result = await service.GetOrder(order.CallOffId);
+
+            Assert.NotNull(result);
+            result.Should().Be(expectedResult);
+        }
+
+        [Test]
+        [InMemoryDbAutoData]
+        public static async Task GetOrderList_ReturnsNull(
+            Guid organisationId,
+            OrderService service)
+        {
+            var result = await service.GetOrderList(organisationId);
+
+            result.Should().BeEmpty();
+        }
+
+        [Test]
+        [InMemoryDbAutoData]
+        public static async Task GetOrderList_ReturnsOrderList(
+            [Frozen] ApplicationDbContext context,
+            Order order,
+            OrderService service)
+        {
+            context.Order.Add(order);
+            await context.SaveChangesAsync();
+
+            var expectedResult = order;
+
+            var result = await service.GetOrderList(order.OrderingParty.Id);
+
+            Assert.NotNull(result);
+            result.Count.Should().Be(1);
+            result.First().Should().Be(expectedResult);
+        }
+
+        [Test]
+        [InMemoryDbAutoData]
+        public static async Task GetOrderSummary_ReturnsNull(
+            CallOffId callOffId,
+            OrderService service)
+        {
+            var result = await service.GetOrderSummary(callOffId);
+
+            result.Should().BeNull();
+        }
+
+        [Test]
+        [InMemoryDbAutoData]
+        public static async Task GetOrderSummary_ReturnsOrder(
+            [Frozen] ApplicationDbContext context,
+            Order order,
+            OrderService service)
+        {
+            context.Order.Add(order);
+            await context.SaveChangesAsync();
+
+            var expectedResult = order;
+
+            var result = await service.GetOrderSummary(order.CallOffId);
+
+            Assert.NotNull(result);
+            result.Should().Be(expectedResult);
+        }
+
+        [Test]
+        [InMemoryDbAutoData]
+        public static async Task GetOrderCompletedStatus_ReturnsNull(
+            CallOffId callOffId,
+            OrderService service)
+        {
+            var result = await service.GetOrderCompletedStatus(callOffId);
+
+            result.Should().BeNull();
+        }
+
+        [Test]
+        [InMemoryDbAutoData]
+        public static async Task GetOrderCompletedStatus_ReturnsOrder(
+            [Frozen] ApplicationDbContext context,
+            Order order,
+            OrderService service)
+        {
+            context.Order.Add(order);
+            await context.SaveChangesAsync();
+
+            var expectedResult = order;
```

Comment:
As above.

Reply:
Updated `expectedResult `and updated `assert`